<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>accountDialog</class>
 <widget class="QDialog" name="accountDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>385</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Glybook ~ Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/icons/icon_final.ico</normaloff>:/icons/icon_final.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">
QRadioButton{
	color:white;
}

QLineEdit {
border: 2px solid #8f8f91;
border-radius: 6px;
   /* background-image: url(:/icons/search.png);
    background-repeat: no-repeat;
    background-position: left;*/
    color: #252424;
    font-family: SegoeUI;
    font-size: 12px;
}

QPushButton {
	color: white;
	border-radius: 8px;
	background-color: rgb(29, 81, 131);
    border-style: outset;
    border-width: 2px;
    border-color:rgb(24, 70, 113);
    padding: 6px;
}
QPushButton:pressed {
    border-style: inset;
}

QPushButton:hover {
	background-color: rgb(23, 66, 106);
    color:white;
    border-style: outset;
}</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>111</width>
     <height>231</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="type_lbl">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;</string>
      </property>
      <property name="text">
       <string>Type of account</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lName_lbl_3">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;</string>
      </property>
      <property name="text">
       <string>Last Name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="fName_lbl_3">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;</string>
      </property>
      <property name="text">
       <string>First Name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="username_lbl_3">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;</string>
      </property>
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="pass_lbl_3">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;</string>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="address_lbl">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;</string>
      </property>
      <property name="text">
       <string>Address</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="phone_lbl_3">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;</string>
      </property>
      <property name="text">
       <string>Phone</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="limit_lbl_3">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;</string>
      </property>
      <property name="text">
       <string>Reservation limit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="lNameEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>120</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="phoneEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>270</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="addressEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>240</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>340</y>
     <width>341</width>
     <height>34</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="allButtonBox_3">
    <item>
     <widget class="QDialogButtonBox" name="dialogButton">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="locale">
       <locale language="English" country="UnitedStates"/>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Close|QDialogButtonBox::Save</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3"/>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="passEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>210</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="resvLimitEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>300</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="fNameEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>150</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="userEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>180</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="adm_RB">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>90</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Administrator</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="sub_RB">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Subscriber</string>
   </property>
  </widget>
  <widget class="QLabel" name="picture">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resource.qrc">:/icons/adherent.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>341</width>
     <height>391</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/icons/fond.jpg);
background-position: left center;</string>
   </property>
  </widget>
  <zorder>graphicsView</zorder>
  <zorder>verticalLayoutWidget</zorder>
  <zorder>lNameEdit</zorder>
  <zorder>phoneEdit</zorder>
  <zorder>addressEdit</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>passEdit</zorder>
  <zorder>resvLimitEdit</zorder>
  <zorder>fNameEdit</zorder>
  <zorder>userEdit</zorder>
  <zorder>adm_RB</zorder>
  <zorder>sub_RB</zorder>
  <zorder>picture</zorder>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
